modules/video/src/video-loader.js

```diff
@@ -0,0 +1,28 @@
+// __VERSION__ is injected by babel-plugin-version-inline
+/* global __VERSION__ */
+import parseVideo from './lib/parsers/parse-video';
+
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+const EXTENSIONS = ['mp4'];
```

Comment:
This probably can support more extensions, like `webm`. We can experiment later.